src/main/cpp/blaze.cc

```diff
@@ -1012,10 +1018,24 @@ static void EnsureCorrectRunningVersion(const StartupOptions &startup_options,
     std::unique_ptr<blaze_util::IFileMtime> mtime(
         blaze_util::CreateFileMtime());
     if (!mtime->SetToNow(blaze_util::Path(startup_options.install_base))) {
-      string err = GetLastErrorString();
-      BAZEL_DIE(blaze_exit_code::LOCAL_ENVIRONMENTAL_ERROR)
-          << "failed to set timestamp on '" << startup_options.install_base
-          << "': " << err;
+      // Ignore permissions errors (i.e. if the install base is not writable):
+      bool raise_error = true;
+
+#if defined(__linux__) || defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__APPLE__) || defined(__CYGWIN__)
```

Comment:
Wasn't sure whether listing out these POSIX-y platforms explicitly was required (or if just using `!defined(_WIN32)` would be sufficient).